src/main/java/com/extendedclip/papi/expansion/server/ServerExpansion.java

```diff
@@ -105,6 +110,36 @@ public Map<String, Object> getDefaults() {
 		return defaults;
 	}
 
+	private @Nullable String getCached(String key, Callable<Integer> callable) {
+		try {
+			return String.valueOf(cache.get(key, callable));
+		} catch (ExecutionException e) {
+			getPlaceholderAPI().getLogger().log(Level.SEVERE, "[server] Could not access cache key " + key, e);
+			return "Error: " + e.getMessage() + " (see console)";
```

Comment:
Oh god no. Please do not return error messages in placeholder results. Just return an empty string or maybe null.

Reply:
Also, depending on how frequent the placeholder is called could this cause spam in the console too, so perhaps also remove the logger line too (or find a way to have a sort of cooldown for the log message sending)

Reply:
Should be resolved with the latest commit.